client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -300,6 +300,9 @@ public class ClientOptions
     @Option(names = "--decimal-data-size", description = "Show data size and rate in base 10 rather than base 2")
     public boolean decimalDataSize;
 
+    @Option(names = "--prefetch-buffer-size", paramLabel = "<prefetch-buffer-size>", defaultValue = "64000", description = "Experimental spooled protocol prefetch buffer size, default: + " + DEFAULT_VALUE)
```

Comment:
let's make it a DataSize, you'll need to register DataSize converter similarly to a Duration one